src/components.jsx/AddComment.jsx

```diff
@@ -31,17 +33,20 @@ export const AddComment = ({ setCurrentComments }) => {
     setCurrentComments((prevComments) => [
       {
         body: commentInput,
-        author: user.username ? user.username : 'Guest',
+        author: user.username ? user.username : "Guest",
         created_at: new Date().toISOString(),
         votes: 0,
         comment_id: 1,
       },
       ...prevComments,
     ]);
-    postComment(commentInput, article_id, user.username);
+    postComment(commentInput, article_id, user.username, setErr);

```

Comment:
if there is an error how would you want this to handle from a user perspective? Should the comment be lost or perhaps repopulated into the comment box in order to try again?